package.json

```diff
@@ -308,6 +308,10 @@
                 "command": "vscode-neovim.delete-char-left-cmdline",
                 "title": "Neovim: Delete char cmdline"
             },
+            {
+                "command": "vscode-neovim.match-cursor-search",
```

Comment:
Thanks for the PR! Please rename to `vscode-neovim.match-cursor-search-cmdline`.